src/main/java/org/takes/http/BkTimeable.java

```diff
@@ -129,7 +61,54 @@ private void check() {
      * @param msec Execution latency
      */
     BkTimeable(final Back back, final long msec) {
-        super(new BkThreads(msec, back));
+        super();
+        this.threads = new ConcurrentHashMap<Thread, Long>(1);
+        this.back = back;
+        this.latency = msec;
     }
 
+    @Override
+    public void start() {
```

Comment:
@sebing I'm curious, why can't we do it all in `run()`? why do we need "start" and "run"?